src/system/timer_data.hh

```diff
@@ -0,0 +1,145 @@
+/*
+ * TimerData.hh
+ *
+ *  Created on: 2. 3. 2015
+ *      Author: jan-hybs
+ */
+
+#ifndef TIMERDATA_HH_
+#define TIMERDATA_HH_
+
+#include "config.h"
+#include <string>
+#include <time.h>
+
+/**
+ * Include either Windows or chrono lib
+ */
+#ifdef FLOW123D_HAVE_TIMER_QUERY_PERFORMANCE_COUNTER
```

Comment:
Is it necessary to make includes here. 
Rather move them to the source file (faster builds).